packages/web-component/src/lib/descope-wc/DescopeWc.ts

```diff
@@ -831,6 +832,25 @@ class DescopeWc extends BaseDescopeWc {
     );
   }
 
+  // handle storing passwords in password managers
+  #handleStoreCredentials(formData = {}) {
+    const idFields = ['externalId', 'email', 'phone'];
+    const passwordFields = ['newPassword', 'password'];
+
+    const id = getFirstNonEmptyValue(formData, idFields);
+    const password = getFirstNonEmptyValue(formData, passwordFields);
+
+    if (id && password) {
```

Comment:
@nirgur , performance wise, isn't it more affective to first check if :
```
const password = getFirstNonEmptyValue(formData, passwordFields);
if (password) {
```

and only if true, do the next check:
```
const id = getFirstNonEmptyValue(formData, idFields);
``` 

??
since this is called on every componant , it could have a big impact no?

Reply:
In general, yes, but It's completely negligible
This code is running only when clicking on next in a flow screen
we are running on 2 keys, don't know if this is something we can even measure
I think we can ignore it